application/src/main/resources/ecc.yml

```diff
@@ -93,6 +93,26 @@ connection:
     ## The default provider will be used unless another is specified.
     ##
     provider: com.ericsson.bss.cassandra.ecchronos.application.providers.AgentJmxConnectionProvider
+    retryPolicy:
+      ## Max number of attempts ecChronos will try to connect with Cassandra.
+      maxAttempts: 5
+      ## Delay use to wait between an attempt and another, this value will be multiplied by the current attempt count powered by two.
+      ## If the current attempt is 4 and the default delay is 5 seconds, so ((4(attempt) x 2) x 5(default delay)) = 40 seconds.
+      ## If the calculated delay is greater than maxDelay, maxDelay will be used instead of the calculated delay.
+      delay: 5
```

Comment:
perhaps this config should be like

```
delay:
  start: 5
  max: 30
  unit: seconds

```
As it is now it is confusing. If all the comments are skipped in the config file, i assume people would expect unit to be applied to all settings under retryPolicy. that is, initialDelay and fixedDelay as well